docs/API/rejection-codes.md

```diff
@@ -0,0 +1,470 @@
+| 1404   | [SRT_REJX_NOTFOUND](#SRT_REJX_NOTFOUND)           | 1.4.2   | Resource specified by `r` and `h` keys cannot be found.                                                        |
+| 1405   | [SRT_REJX_BAD_MODE](#SRT_REJX_BAD_MODE)           | 1.4.2   | Mode specified in the `m` key in StreamID is not supported for this request.                                   |
+| 1406   | [SRT_REJX_UNACCEPTABLE](#SRT_REJX_UNACCEPTABLE)   | 1.4.2   | Unavailable parameters in `SocketID`, or `m=publish` data format not supported.                                |
+| 1409   | [SRT_REJX_CONFLICT](#SRT_REJX_CONFLICT)           | 1.4.2   | Resource specified by `r` and `h` keys is locked for modification.                                             | 
+| 1415   | [SRT_REJX_NOTSUP_MEDIA](#SRT_REJX_NOTSUP_MEDIA)   | 1.4.2   | Media type  not supported by the application.                                                                  |
+| 1423   | [SRT_REJX_LOCKED](#SRT_REJX_LOCKED)               | 1.4.2   | Resource is locked against any access.                                                                         |
+| 1424   | [SRT_REJX_FAILED_DEPEND](#SRT_REJX_FAILED_DEPEND) | 1.4.2   | Dependent entity for the request is not present.                                                               |
+| 1500   | [SRT_REJX_ISE](#SRT_REJX_ISE)                     | 1.4.2   | Internal server error.                                                                                         |
+| 1501   | [SRT_REJX_UNIMPLEMENTED](#SRT_REJX_UNIMPLEMENTED) | 1.4.2   | Request not supported by current version of the service.                                                       |
+| 1502   | [SRT_REJX_GW](#SRT_REJX_GW)                       | 1.4.2   | Target endpoint rejected connection from gateway server                                                        |
+| 1503   | [SRT_REJX_DOWN](#SRT_REJX_DOWN)                   | 1.4.2   | Service is down for maintenance.                                                                               |
+| 1505   | [SRT_REJX_VERSION](#SRT_REJX_VERSION)             | 1.4.2   | SRT application version not supported.                                                                             |
+| 1507   | [SRT_REJX_NOROOM](#SRT_REJX_NOROOM)               | 1.4.2   | Data stream cannot be archived due to lack of storage space.                                                   |
+|        | <img width=250px height=1px/>                     |         |                                                                                                                |
+
+
+**NOTE**: SRT rejection codes follow this prefix convention:
+
+  - `SRT_REJ`: standard rejection codes from SRT API functions (0 - 99)
+  - `SRT_REJC`: mark the border values between ranges.
+  - `SRT_REJX`: extended rejection codes (400 – 599)  :warning: *@max is this the correct range (see comments above)?*
```

Comment:
See inline comment.

Reply:
Should be simply "code values above 1000", if you speak about the `SRT_REJX` prefix.